lib/recart-nacelle-plugin.js

```diff
@@ -1,5 +1,5 @@
 const { getMetafieldsForCheckout, saveCheckoutId } = require('./checkout')
-const { saveCart } = require('./cart')
+const cartLib = require('./cart')
```

Comment:
Had to change to be able to stub for tests.

I am not satisfied with the naming here. I didn't want to use cart as it would conflict with the nacelle cart.